.github/workflows/master.yml

```diff
@@ -24,4 +24,4 @@ jobs:
       run: |
         export MAVEN_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=512m -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN"
         export MAVEN_CLI_OPTS="--no-transfer-progress"
-        ./build/mvn $MAVEN_CLI_OPTS -DskipTests package
+        ./build/mvn $MAVEN_CLI_OPTS -DskipTests -Djava.version=${{ matrix.java }} package
```

Comment:
Ur, for GitHub workflow, we aims to do build-test with a single JDK. So, this is not required, isn't it?
Technically, when we build with JDK8 and test them on JDK11, we use `-Djava.version=11` during testing.

Reply:
When we run mvn, its java.version property is 1.8, if not specified. Even we use matrix, I think mvn should still build with 1.8?

Reply:
Oh, got it. You want to use `11` for `source` and `target` completely, right?

Reply:
Actually, that one is my original intention. But this PR is better because it aims to move forward from it.

Reply:
Previously, I wanted to use JDK11 for building and testing with `-release 8` option which means `source=8`, `target=8`, `bootclass=8`. It's an intermediate status.

Reply:
It's interesting. Is there a way to have all of those combinations?

Reply:
yeah, I think java.version controls source and target.